builds/service-onboarding-build-pack/Jenkinsfile

```diff
@@ -98,6 +99,21 @@ node  {
 						}
 						break
 
+					case "sls-app":
```

Comment:
`runtime` is not a required field for sls_app services (example [sls yml](https://github.com/serverless/examples/blob/master/aws-node-single-page-app-via-cloudfront/serverless.yml)). In this case, we'll need provide a runtime agnostic template to the user.

Reply:
Well, I believe the runtime variable is still really relevant for sls-app because of the serverless.com framework limitation: https://forum.serverless.com/t/one-service-multiple-functions-multiple-runtimes/4064 . So far as I understand the service's runtime for all the functions included into the service must be the same as long as they belong to the same serverless.yml. Thus I am not too sure how we can forge a truly runtime agnostic template for a developer

Reply:
Hmm, that's making sense. I think we should still support use cases with no runtimes (developer can create a static site using s3/cloudfront). This is a TODO for us but for this release, we'll keep this simple.  

Looks ok for now.